src/finiteVolume/processes/twoPhaseEulerAdvection.cpp

```diff
@@ -203,29 +203,99 @@ ablate::finiteVolume::processes::TwoPhaseEulerAdvection::TwoPhaseEulerAdvection(
       fluxCalculatorLiquidLiquid(std::move(fluxCalculatorLiquidLiquid)) {}
 
 void ablate::finiteVolume::processes::TwoPhaseEulerAdvection::Setup(ablate::finiteVolume::FiniteVolumeSolver &flow) {
+    // Before each step, compute the alpha
+    auto multiphasePreStage = std::bind(&ablate::finiteVolume::processes::TwoPhaseEulerAdvection::MultiphaseFlowPreStage, this, std::placeholders::_1, std::placeholders::_2, std::placeholders::_3);
+    flow.RegisterPreStage(multiphasePreStage);
+
     // Create the decoder based upon the eoses
     decoder = CreateTwoPhaseDecoder(flow.GetSubDomain().GetDimensions(), eosGas, eosLiquid);
 
     // Currently, no option for species advection
-    flow.RegisterRHSFunction(CompressibleFlowComputeEulerFlux, this, "euler", {"densityVF", "euler"}, {});
-    flow.RegisterRHSFunction(CompressibleFlowComputeVFFlux, this, "densityVF", {"densityVF", "euler"}, {});
+    flow.RegisterRHSFunction(CompressibleFlowComputeEulerFlux, this, "euler", {"volumeFraction", "densityVF", "euler"}, {});
```

Comment:
Would you mind changing these to a const variable like CompressibleFlowFields::EULER_FIELD?